commands/version/index.js

```diff
@@ -72,7 +72,9 @@ class VersionCommand extends Command {
 
     this.gitRemote = gitRemote;
     this.tagPrefix = tagVersionPrefix;
-    this.commitAndTag = gitTagVersion;
+    this.shouldCommit = this.options.gitCommit && gitTagVersion;
+    this.shouldTag = this.options.gitTag && gitTagVersion;
```

Comment:
How exactly does one tag a release commit that doesn't exist? I'm very confused by this.

Reply:
Thank you for the review. Your concern is valid. I've reviewed the logic and discussed it in a team and I'll update the pull request based on your comments.

Reply:
The functionality should indeed not tag the release commit. It is indeed not possible as it does not exist. However, it can tag the last commit, no?